### PR TITLE
apps/examples/rtc : Add rtc accuracy test example

### PR DESCRIPTION
**Pull request by jeongchanKim**
This example runs forever with printing the log on every 5mins.
With this log, we can check the time difference which rtc runs well or not.

### CODE REVIEW

File: apps/examples/rtc/rtc_main.c [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+ ****************************************************************************/
+
+#ifdef CONFIG_BUILD_KERNEL
+int main(int argc, FAR char *argv[])
+#else
+int rtc_main(int argc, char *argv[])
+#endif
+{
+	printf("=== RTC Accuracy Test ===\n");
+	printf(" * This test will run forever !!! *\n");
```
Comment (sunghan-chang) [inline apps/examples/rtc/rtc_main.c:147]: Let's support "stop" like stack monitor.

File: apps/examples/rtc/rtc_main.c [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+	/* Initialize the RTC time as 0 */
+	ret = ioctl(fd, RTC_SET_TIME, (unsigned long)&prev_time);
+	if (ret < 0) {
+		printf("ERROR : Fail to set RTC init time.\n");
+		close(fd);
+		return NULL;
+	}
+
+	sched_lock();
+	while (rtctest_started) {
```
Comment (sunghan-chang) [inline apps/examples/rtc/rtc_main.c:123]: When disable_signal config is enabled, this function is called with this variable 0. Is it ok?

File: apps/examples/rtc/rtc_main.c [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+ * rtc_main
+ ****************************************************************************/
+
+#ifdef CONFIG_BUILD_KERNEL
+int main(int argc, FAR char *argv[])
+#else
+int rtc_main(int argc, char *argv[])
+#endif
+{
+#ifndef CONFIG_DISABLE_SIGNALS
```
Comment (sunghan-chang) [inline apps/examples/rtc/rtc_main.c:166]: Could you let me know why you cover with disable_signals config? not disable_pthread?

File: apps/examples/rtc/rtc_main.c (hunk 2 above)
Comment (jeongchanKim) [reply apps/examples/rtc/rtc_main.c:123]: Modified. Thanks.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**